squire_lib/src/accounts.rs

```diff
@@ -63,19 +76,33 @@ pub struct OrganizationAccount {
 }
 
 impl SquireAccount {
-    /// Creates a new account object
-    pub fn new(user_name: String, display_name: String) -> Self {
-        Self {
-            user_name,
+    pub fn create(user_name: String, display_name: String, permissions: SharingPermissions) -> Self {
```

Comment:
This is really a stylist thing, but the primary method used for object creation is often called `new`. I believe this is a lint that clippy will suggest and is what the rest of the library uses.
```suggestion
    pub fn new(user_name: String, display_name: String, permissions: SharingPermissions) -> Self {
```